SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -363,13 +389,6 @@ void USpatialWorkerConnection::FinishSetupConnectionConfig(const FURL& URL, bool
 			ReceptionistConfig.UseExternalIp = !UseExternalIpOption.Equals(TEXT("false"), ESearchCase::IgnoreCase);
 		}
 	}
-	else
```

Comment:
Is it intentional, that we don't use locator anymore?

Reply:
we do still use the locator. the logic simply has changed a bit.  the locator part is in in line 366 -372

Reply:
Oh, right! Resolving.